.github/workflows/samples.yml

```
@@ -0,0 +1,45 @@
+# This workflow runs a `deno check` against slack sample apps
+name: Deno check
```

Let's rename this, as this will be contributor-facing, so let's come up with a concise name that makes it clear what this test is doing.

```suggestion
name: Samples Integration Type-checking
```